### PR TITLE
Fix 771 Only resize if menu open

### PR DESCRIPTION
**Pull request by mlh758**
The fix was to not resize the menu if the widget isn't open.

I also moved the input binds to their own method, a small refactor.

@SteveTheTechie let me know if this might break any of your new code.

### CODE REVIEW

**Comment by SteveTheTechie** [comment]
Give me a chance to try to reproduce the issue w/ options that are valid.   The header option that was posted in the original issue is invalid.

File: src/jquery.multiselect.filter.js [hunk 1]
```diff
@@ -94,7 +94,43 @@
           type: "search"
         })
         .css({  width: (/\d/.test(opts.width) ? opts.width + 'px' : null) })
```
Comment (SteveTheTechie) [inline src/jquery.multiselect.filter.js]: Please add a semi at the end here.

File: src/jquery.multiselect.filter.js [hunk 2]
```diff
@@ -213,7 +220,7 @@
       if (term) {
          this._trigger('filter', e, filteredInputs);
       }
-      if (!this.instance.options.listbox) {
+      if (!this.instance.options.listbox && this.instance._isOpen) {
```
Comment (SteveTheTechie) [inline src/jquery.multiselect.filter.js]: I think this is fine.

**Comment by SteveTheTechie** [comment]
@mlh758 I took a look and I think this is fine.

File: src/jquery.multiselect.filter.js (hunk 1 above)
Comment (mlh758) [reply src/jquery.multiselect.filter.js]: Added it.

**Pull request merged**